_includes/map.html

```diff
@@ -0,0 +1,50 @@
+<script>
+
+// Store a list of rses, and groups to add to map
+var RSEs = Array()
+var groups = Array()
+
+// First add authors, optionally with image and institution
+{% for author in site.data.authors %}{% if author.coords %}
+var marker = L.marker([{{ author.coords[0] }}, {{ author.coords[1] }}]);
+marker.bindPopup("Research Software Engineer<br>{% if author.image %}<img src='{{ author.image }}' style='height:30px; float:right'>{% endif %}<a href='{{ author.url }}'>{{ author.name }}{% if author.institution %}<br>{{ author.institution }}{% endif %}</a>");
+RSEs.push(marker);{% endif %}{% endfor %}
+
+// Add them to a layer group
+var rseLayer = L.layerGroup(RSEs);
+
+// Next add groups, and RSEs without feeds
+{% for entry in site.data.map %}{% if entry.coords %}
+var marker = L.marker([{{ entry.coords[0] }}, {{ entry.coords[1] }}]);
+marker.bindPopup("RSE Group<br>{% if entry.image %}<img src='{{ entry.image }}' style='height:30px; float:right'>{% endif %}<a href='{{ entry.url }}'>{{ entry.name }}{% if entry.institution %}<br>{{ entry.institution }}{% endif %}</a>");
```

Comment:
If I'm reading this correctly, you've hardcoded in the string `RSE Group`, even in the case of an individual RSE without a feed.  Can we add a way to distinguish the two?

Reply:
I just said the same thing!  Seems we're on the same page. :)